tidb-cloud/tidb-cloud-quickstart.md

```diff
@@ -236,3 +145,5 @@ When the process of importing data is completed, you can start to run some queri
     ORDER BY count ASC
     LIMIT 10;
     ```
+
+For more details on TiDB SQL usage, see [Explore SQL with TiDB](/basic-sql-operations.md).
```

Comment:
Could we add a sentence here to guide users to the https://docs.pingcap.com/tidbcloud/create-tidb-cluster doc for creating Dedicated Tier cluster?